build/lineoperations.m4

```diff
@@ -0,0 +1,10 @@
+AC_DEFUN([GP_CHECK_LINEOPERATIONS],
+[
+    GP_ARG_DISABLE([LineOperations], [auto])
+    GP_CHECK_PLUGIN_GTK2_ONLY([LineOperations])
```

Comment:
Unless you know why you wouldn't want to support GTK3, you should remove this and try.  Actually, as I read your code nothing should stop you from supporting both GTK2 and GTK3 without a change.
So i'd suggest to drop this GTK2 enforcement